app/jobs/remediation_module.rb

```diff
@@ -5,10 +5,24 @@ module RemediationModule
   OUTPUT_POLLING_TIMEOUT = 3600 # The default 1-hour timeout is also arbitrary and should probably be adjusted.
   PRESIGNED_URL_EXPIRES_IN = 84_000
```

Comment:
I think we should move this module to a `concerns/` directory in `jobs/`.  That's the idiomatic Rails structure.